src/Core/Prefab/GlobeView.js

```diff
@@ -163,7 +163,7 @@ function GlobeView(viewerDiv, coordCarto, options) {
             this._latestUpdateStartingLevel = commonAncestor.level;
             return [commonAncestor];
         } else {
-            return [];
+            return layer.level0Nodes;
```

Comment:
Nice. But the purpose is defeated by https://github.com/iTowns/itowns/pull/392/files#diff-caf5227be4f9e2f283d82df32a678e4bL150

The above loop assumed that `changeSources` content were all `TileMesh`.
With this PR this is no longer the case (which is a good thing), but you need add a safeguard in the loop for this case.

Options:
- add `if (source.isCamera) { ... }` but this will break as soon as a non camera object is used
- add a `isTileMesh` property ?
- test for `isMesh` and `node.level` existence?